Dockerfile

```diff
@@ -135,14 +148,12 @@ RUN luarocks install luacheck
 # NPM setup
 # FIXME: we should use package.json from coala
 RUN npm install -g \
-  alex \
```

Comment:
Ya, I did see that, but it is only one test that fails, and I doubt that `alex` is installed on this rather empty openSUSE image.  And fa70e924 added checks for this , and those tests pass.
A neater solution would be to disable only the one failing test.